Updated logic to determine if a data set is a PDS & fixed non existent source data set bug (#2448)
zowe/zowe-cli: packages/zosfiles/CHANGELOG.md

## Recent Changes

> -BugFix: When using the copy command with a non existent source data set, an unclear error was displayed to the user. Now, the user is shown a user-friendly error. [#2447] (https://github.com/zowe/zowe-cli/issues/2447).

Reviewer: ```suggestion
-BugFix: When using the copy command with a non-existent source data set, an unclear error message displayed to the user. Now, a more user-friendly message is provided. [#2447] (https://github.com/zowe/zowe-cli/issues/2447).
```

> -BugFix: The updated logic to determine whether a data set is a partitioned data set now ensures that `dsorgs` of `PO-E`, `PO-L`, `PO-U` are handled correctly. [#2390] (https://github.com/zowe/zowe-cli/issues/2390).

Reviewer: We should add adjectives to describe what `dsorgs` and `PO-E`, `PO-L`, `PO-U` are.
